Olfeo/_meta/manifest.yml

```
@@ -0,0 +1,5 @@
+uuid: f74ed0e8-a4e2-43e3-9bfc-a846c2c7d612
+name: Olfeo secure web gateway
```

This manifest defines the information about the module. Usually, a module corresponds to a vendor.
I suggest using the name of the Vendor
```suggestion
name: Olfeo
```